src/devtools/requests.ts

```diff
@@ -0,0 +1,119 @@
+import * as Rx from 'rxjs'
+
+import type Protocol from 'devtools-protocol'
+
+type TargetInfo = chrome.debugger.TargetInfo
+
+export type Debuggee = chrome.debugger.Debuggee
+export type Request = Protocol.Network.Request
+export type RequestId = Protocol.Network.RequestId
+export type RequestPausedEvent = Protocol.Fetch.RequestPausedEvent
+export type PausedEvent = [
+  debuggee: Debuggee,
+  method: string,
+  params?: RequestPausedEvent | undefined
+]
+
+export type InterceptorAPI = {
+  requests$: Rx.Observable<PausedEvent | null>
+  pause$: Rx.Subject<boolean>
+  continue$: Rx.Subject<{ requestId: RequestId; request: Request }>
+  forceDetach$: Rx.Subject<boolean>
+}
```

Comment:
Nice API design!

What's the `$`, I wonder? Is it RxJS convention to designate an Observable or other RxJS type?

Reply:
`$` is the convention for Observables